calico/getting-started/kubernetes/managed-public-cloud/aks.md

```diff
@@ -9,15 +9,91 @@ Enable {{site.prodname}} in AKS managed Kubernetes service.
 
 ### Value
 
-AKS has built-in support for {{site.prodname}}, providing a robust implementation of the full Kubernetes Network Policy API. AKS users wanting to go beyond Kubernetes network policy capabilities can make full use of the {{site.prodname}} Network Policy API. 
+AKS has built-in support for {{site.prodname}}, providing a robust implementation of the full Kubernetes Network Policy API. AKS users wanting to go beyond Kubernetes network policy capabilities can make full use of the {{site.prodname}} Network Policy API.
+
+You can also use {{site.prodname}} for networking on AKS in place of the default Azure VPC networking. This allows you to take advantage of the full set of {{site.prodname}} networking features.
 
 ### How to
 
-To enable {{site.prodname}} network policy enforcement, follow these step-by-step instructions: {% include open-new-window.html text='Create an AKS cluster and enable network policy' url='https://docs.microsoft.com/en-us/azure/aks/use-network-policies' %}.
+#### Install AKS with {{site.prodname}} for network policy
 
 The geeky details of what you get:
 {% include geek-details.html details='Policy:Calico,IPAM:Azure,CNI:Azure,Overlay:No,Routing:VPC Native,Datastore:Kubernetes' %}
 
+To enable {{site.prodname}} network policy enforcement, follow these step-by-step instructions: {% include open-new-window.html text='Create an AKS cluster and enable network policy' url='https://docs.microsoft.com/en-us/azure/aks/use-network-policies' %}.
+
+#### Install AKS with {{site.prodname}} networking
+
+The geeky details of what you get:
+{% include geek-details.html details='Policy:Calico,IPAM:Calico,CNI:Calico,Overlay:VXLAN,Routing:Calico,Datastore:Kubernetes' %}
+
+1. Create an Azure AKS cluster with no Kubernetes CNI pre-installed. Please refer to [Bring your own CNI with AKS](https://docs.microsoft.com/en-us/azure/aks/use-byo-cni?tabs=azure-cli) for details.
+
+  ```bash
```

Comment:
These commands just duplicate the stuff in the Azure link above.  Do we want them reproduced here?
Pros: easier for beginners
Cons: will be out of date if/when msft update their instructions.

Reply:
I'm in favour of duplicate the commands here as I don't think AKS will update their instructions frequently and our e2e should pick up any issue even they do.

Reply:
If we're duplicating it here for beginners, can I suggest we actually put values in for location, cluster name, etc.  

More advanced beginners will know to substitute the values, and it'll "just work" for complete beginners.

Reply:
Done